static/docs/commands-reference/config.md

```diff
@@ -41,156 +45,163 @@ share through Git.
 
 * `-v`, `--verbose` - displays detailed tracing information.
 
-## core
+## Available setting names and sections
+
+These are the `name` parameters that can be used with `dvc config`, or the
+sections in the project config file (`.dvc/config`).
+
+### remote
+
+These are sections in the config file that describe particular remotes. See `dvc
+remote` for more info.
+
+### core
+
+This is the main section with the general settings:
```

Comment:
let's double check. I'm pretty sure that we now have a whole bunch of new options.

Reply:
There's tons of config sections listed in https://github.com/iterative/dvc/blob/7ceaf88c54c90d53e64c1af2fa78b5b9bd4667a0/dvc/config.py#L139-L274 but I don't know what they do. I'll ask on Discord.

Reply:
Added all `core` and `cache` options but still not finished with all the other sections. Will lky